### PR TITLE
[Catalog] Component descriptions update.

### PR DESCRIPTION
**Pull request by mohammadcazig**
Pivotal: https://www.pivotaltracker.com/story/show/156939588

List of component with updated description:
App Bar,
Bottom App Bar,
Bottom Navigation,
Bottom Sheet,
Buttons,
Cards,
Chips,
Dialogs,
Progress View,
Slider,
Snackbar,
Tab Bar,
Text Fields,

List of component that do not have an existing updates from the specs:
Activity Indicator,
Animation Timing,
Button Bar,
Collection Cells,
Collections,
Color Scheme,
Feature Highlight,
Flexible Header,
Header Stack View,
Ink,
Masked Transition,
Navigation Bar,
Page Control,
Palettes,
Shadow,
Themes,
Typography and Fonts,

screen shot 2018-04-25 at 1 42 36 pm
screen shot 2018-04-25 at 1 42 44 pm

Note for more than 3 lines we allow scrolling on description:
I opened a pivotal task to address this issue and makre resizable description headers:
https://www.pivotaltracker.com/story/show/157096487
screen shot 2018-04-25 at 1 42 25 pm

### CODE REVIEW

File: components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m [hunk 1]
```diff
@@ -33,8 +33,8 @@ + (NSArray *)catalogBreadcrumbs {
 }
 
 + (NSString *)catalogDescription {
-  return @"Buttons is a collection of Material Design buttons, including a text button, a contained"
-          " button and a floating action button.";
+  return @"Buttons allow users to take actions, and make choices, with a single tap.\nA floating"
```
Comment (jverkoey) [inline components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m:36]: Does this need a newline?

File: components/Cards/examples/CardsCollectionTintingExample.m [hunk 2]
```diff
@@ -116,7 +116,7 @@ + (BOOL)catalogIsPrimaryDemo {
 }
 
 + (NSString *)catalogDescription {
-  return @"Material Cards.";
+  return @"Cards contain content and actions about a single subject.";
 }
```
Comment (jverkoey) [inline components/Cards/examples/CardsCollectionTintingExample.m]: Delete this description. Descriptions are only necessary for primary demos.

File: components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m [hunk 3]
```diff
@@ -23,8 +23,7 @@ + (NSArray *)catalogBreadcrumbs {
 }
 
 + (NSString *)catalogDescription {
-  return @"The bottom app bar is a bar docked at the bottom of the screen that has a floating "
-         @"action button and can provide navigation.";
+  return @"A bottom app bar displays navigation and key actions at the bottom of mobile screens.";
```
Comment (jverkoey) [inline components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m]: "bottom of the screen"

**Review by jverkoey — DISMISSED** [review]

File: components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m (hunk 1 above)
Comment (mohammadcazig) [reply components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m:36]: In our specs they are even separate components so I thought it should be clear that this is a separate specific thing we say. I felt like it's a context change so it should be a new line but let me know if you think it makes sense to be in the same line.

File: components/Cards/examples/CardsCollectionTintingExample.m (hunk 2 above)
Comment (mohammadcazig) [reply components/Cards/examples/CardsCollectionTintingExample.m]: Done.

File: components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m (hunk 3 above)
Comment (mohammadcazig) [reply components/BottomAppBar/examples/supplemental/BottomAppBarTypicalUseSupplemental.m]: Done.

File: components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m (hunk 1 above)
Comment (jverkoey) [reply components/Buttons/examples/supplemental/ButtonsTypicalUseSupplemental.m:36]: I'm not sure we gain much from having one - may be cleaner just to remove.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**